omnisharptest/omnisharpIntegrationTests/diagnostics.integration.test.ts

```diff
@@ -53,10 +56,12 @@ describeIfNotGenerator(`DiagnosticProvider: ${testAssetWorkspace.description}`,
         virtualRazorFileUri = vscode.Uri.file(razorFileUri.fsPath + '__virtual.cs');
     });
 
+    afterAll(async () => {
+        await testAssetWorkspace.cleanupWorkspace();
+    });
+
     describeIfRazor('razor workspace', () => {
         beforeAll(async function () {
-            const activation = await activateCSharpExtension();
-            await testAssetWorkspace.restore();
```

Comment:
I believe the workspace cleanup and all this activation / restarting was causing a bunch of file change events to get fired while the server was stopping.  By doing the restore and cleanup once for the entire suite